front/src/store/modules/movies.store.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+export default {
+  state: () => ({
+    movies: []
+  }),
+  mutations: {
+    initMovies (state, serverData) {
+      state.movies = serverData;
+    }
+  },
+  getters: {
+    getMoviesList (state) {
+      return state.movies;
```

Comment:
are you sure that it is really necessary to create getter. it seems it is enough state. it have sense for sorting, filtering and other things. as i remember in documentation there are some thoughts about it